toolchain/check/testdata/interop/cpp/function/arithmetic_types_direct.carbon

```diff
@@ -106,52 +106,6 @@ fn F() {
   //@dump-sem-ir-end
 }
 
-// ============================================================================
-// unsigned int param
-// ============================================================================
-
-// --- unsigned_int_param.h
-
-auto foo(unsigned int a) -> void;
```

Comment:
Presumably we should eventually classify `unsigned int` as direct rather than bridged? I think `u32`, like `i32`, has a simple ABI rule across the targets we care about.

Reply:
Let's not add more to this PR. We can switch to passing u32 / u64 directly later.

Reply:
SGTM. Not taking action here.